pipelines/build/common/create_job_from_template.groovy

```diff
@@ -47,6 +52,7 @@ pipelineJob("$buildFolder/$JOB_NAME") {
                 }
             }
             scriptPath("${SCRIPT_PATH}")
+            lightweight(lightWeight)
```

Comment:
It's odd to have a function and variable with that same name.  Maybe one of these is `isLightweight`.

Reply:
Good idea, updated.